webpack/scenes/Tasks/TaskActions.js

```diff
@@ -8,31 +8,45 @@ import {
   TASK_BULK_SEARCH_REQUEST,
   TASK_BULK_SEARCH_SUCCESS,
   TASK_BULK_SEARCH_FAILURE,
+  RESET_TASKS,
 } from './TaskConstants';
 
 export const bulkSearch = (extendedParams = {}) => (dispatch) => {
+  const params = {
+    search: Object.entries(propsToSnakeCase(extendedParams))
+      .map((item) => {
+        if (item[0] === 'action') {
+          return `${item[0]}~${item[1]}`;
+        }
+        return `${item[0]}=${item[1]}`;
+      })
+      .join(' and '),
+  };
+
+
   const onBulkSearchSuccess = ({ data }) => dispatch({
     type: TASK_BULK_SEARCH_SUCCESS,
     response: data,
   });
-
+  
   const onBulkSearchFailure = result => dispatch({
     type: TASK_BULK_SEARCH_FAILURE,
     result,
   });
 
   dispatch({ type: TASK_BULK_SEARCH_REQUEST });
-
-  const params = {
-    searches: [{ ...propsToSnakeCase(extendedParams) }],
-  };
-
   return api
-    .post('/tasks/bulk_search', params)
+    .get('/tasks', {}, params)
```

Comment:
Instead of `bulk_search` API endpoind, I used the regular search in order to filter the organization.